presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -4700,6 +4700,12 @@ public void testAccessControl()
         assertAccessDenied("SELECT name AS my_alias FROM nation", "Cannot select from columns \\[name\\] in table .*.nation.*", privilege("name", SELECT_COLUMN));
         assertAccessDenied("SHOW CREATE TABLE orders", "Cannot show create table for .*.orders.*", privilege("orders", SHOW_CREATE_TABLE));
         assertAccessAllowed("SHOW CREATE TABLE lineitem", privilege("orders", SHOW_CREATE_TABLE));
+        assertAccessAllowed("SHOW STATS FOR lineitem");
+        assertAccessAllowed("SHOW STATS FOR lineitem", privilege("orders", SELECT_COLUMN));
```

Comment:
how `SHOW STATS FOR lineitem` is going to work when user has access only to subset of columns? Is it only a sugar for `SHOW STATS FOR (SELECT * FROM lineitem)`? And I should expect a failure that user is not allowed to see some columns, or it should just filter only visible columns?

Reply:
I know. It is a syntax sugar. I fine with that, but please add a test.

Reply:
It will fail when user has no access to a subset of columns

Reply:
Can you please add a test for that?

Reply:
added